adblock-lean

```diff
@@ -490,14 +498,12 @@ generate_preprocessed_blocklist_file_parts()
 			then
 				log_msg "Processing of new blocklist file part from: ${blocklist_url} succeeded (downloaded file size: ${blocklist_file_part_size_human}; sanitized line count: $(int2human ${blocklist_file_part_line_count}))."
 
-				rm -f /var/run/adblock-lean/rogue_element
-
 				preprocessed_blocklist_line_count=$(( preprocessed_blocklist_line_count + blocklist_file_part_line_count ))
 				blocklist_id=$((blocklist_id+1))
 				continue 2
 			else
+				rm -f "/var/run/adblock-lean/blocklist.${blocklist_id}.gz"
 				log_msg -err "Downloaded blocklist file part line count: $(int2human ${blocklist_file_part_line_count}) less than configured minimum: $(int2human ${min_blocklist_file_part_line_count})."
-				break
```

Comment:
Doesn't this simply avoid retrying the download when we know that the download completed and the downloaded blocklist file part simply doesn't have enough lines in it?

Reply:
See: https://forum.openwrt.org/t/optimization-of-adblock-lean/203108/423